ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -94,6 +93,19 @@ Rectangle {
 
     color: Style.current.transparent
 
+    QtObject {
+        id: d
+
+        //mentions helper properties
+        property string copiedTextPlain: ""
+        property string copiedTextFormatted: ""
+        property int copyTextStart: 0
+
+        // set to true when pasted text comes from this component (was copied within this component)
+        property bool internalPaste: false
+    }
+
+
     function calculateExtraHeightFactor() {
         const factor = (messageInputField.length / 500) + 1;
         return (factor > 5) ? 5 : factor;
```

Comment:
Can you please also put 500 and 5 magic numbers to d QtObject?
```Math.min(maxFactor, factor?)```

Reply:
`calculateExtraHeightFactor` is not used, so I just removed it